FAQ addition (#25)
Quaver/Quaver.Wiki: FAQ/en.md

# Frequently Asked Questions

## General

> 

Reviewer: More questions to add:

1. What is Quaver?
2. How is Quaver different from other games?
3. What platforms will Quaver be released on?

Author: Added all 3.

> ### What state is Quaver currently in?

Reviewer: Rename this question to `How far in development is Quaver?`

Author: Done

> Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.

Reviewer: Reword this to: `Quaver is still in early development and is currently in closed alpha. `

Author: Done

### Where can I sign up for the Quaver Alpha?

> We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.

Reviewer: `Registrations for the closed alpha occur in rounds. Each round of sign-ups are announced before they open.`

Author: Applied

### Can I play without being part of the Quaver Alpha?

> Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features.

Reviewer: `...won’t be able to experience any of the online features including online rankings, map uploading, and other community-based features such as in-game chat.`

Author: Applied

### Does this mean I can’t download maps from the Quaver website?

> That is correct, however, Adri (a member of the Quaver team) offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of map packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.

Reviewer: You cannot download maps from the Quaver website, however you can still download packs via a [mirror](https://rhythmgamers.net/pack/).

Author: Applied

### Is there an estimate for the public release date?

We do not have an estimate for the public release date yet.

## Gameplay

> ### What key modes does Quaver support?

Reviewer: `What game modes does Quaver support?`

Author: Applied

> Quaver currently only supports 4 keys and 7 keys.

Reviewer: Quaver currently only supports 4 keys (4k) and 7 keys (7k).

Author: Applied

### Can I import maps from other games?

> Yes, Quaver currently supports .osu and .sm files and we are working on supporting more file formats.

Reviewer: `Yes, Quaver currently supports .osz and .sm files. To import them, drag the files into the game window.`.

Author: Applied

### I have a lot of maps in an external game, can I import them all at once?

It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.

### Can I import my skin from another game?

> Yes, you can download a converter by Adri [here](https://rhythmgamers.net/QBC/) and a tutorial can be found [here](https://www.youtube.com/watch?v=pWeLbx48NVI).

Reviewer: The game doesn't support skin conversions out of the box, however you can download a .osk skin converter [here](https://rhythmgamers.net/QBC/) along with a [tutorial](https://www.youtube.com/watch?v=pWeLbx48NVI).

Author: Applied

### I found a bug, where can I report it?

> Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.

Reviewer: `It doesn't matter how minor you think the issue is.` remove this

Author: Done

### I have a feature request, where can I post it?

Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!

## Troubleshooting

### The game doesn’t launch

Make sure Steam is running, as Steam is required in order to run Quaver. This also applies to the offline build!

### I can’t upload my mapset using the editor

> If your mapset fails to upload using the client, you can upload it manually on the [Upload Mapset](/upload/mapset/) page.

Reviewer: Link is broken

Author: Changed the link to a full quavergame.com link

# Frequently Asked Questions

## General

### What state is Quaver currently in?

Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.

### Where can I sign up for the Quaver Alpha?

We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.

### Can I play without being part of the Quaver Alpha?

Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features.

### Does this mean I can’t download maps from the Quaver website?

> That is correct, however, Adri; a member of Quaver offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of Map Packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.

Reviewer 2: This `;` does not belong here.
```suggestion
That is correct, however, Adri (a member of the Quaver team) offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of map packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.
```

> ### Is there an estimate for the public release?

Reviewer 2: ```suggestion
### Is there an estimate for the public release date?
```

> We do not have an estimate for a public release yet.

Reviewer 2: ```suggestion
We do not have an estimate for the public release date yet.
```

## Gameplay

### What key modes does Quaver support?

Quaver currently only supports 4 keys and 7 keys.

### Can I import maps from other games?

> Yes, Quaver currently supports .osu and .sm files and we are working on increasing the amount of files supported in the near future.

Reviewer 2: ```suggestion
Yes, Quaver currently supports .osu and .sm files and we are working on supporting more file formats.
```

### I have a lot of maps in an external game, can I import them all at once?

> It is possible to import all of your maps at once by linking the games “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.

Reviewer 2: ```suggestion
It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.
```

### Can I import my skin from another game?

Yes, you can download a converter by Adri [here](https://rhythmgamers.net/QBC/) and a tutorial can be found [here](https://www.youtube.com/watch?v=pWeLbx48NVI).

### I found a bug, where can I report it?

> Any bug reports should be posted on our [Github](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but make sure to check for already existing threads to avoid duplication.

Reviewer 2: ```suggestion
Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.
```

### I have a feature request, where can I post it?

> Feature requests belong, just like bug reports, on our [Github](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!

Reviewer 2: ```suggestion
Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!
```

## Troubleshooting

### The game doesn’t launch

> Make sure Steam is running, as Steam is required in order to launch Quaver. This also applies to the offline build!

Reviewer 2: ```suggestion
Make sure Steam is running, as Steam is required in order to run Quaver. This also applies to the offline build!
```

### The game is stuck on the welcome screen

> If the game doesn’t advance from the welcome screen, there might be a Steam-wide issue. Try again a bit later if this occurs.

Reviewer 2: This one is already fixed in the next update, so I'm not sure if this point should be kept.

Author: I guess we can remove it then.

# Frequently Asked Questions

## General

### What is Quaver?

> Quaver is a community-driven rhythm game, aiming to bring VSGRs to the next level. The game is open to players of all skill levels, casual and seasoned.

Reviewer: Quaver is a community-driven and open source competitive rhythm game. It is inspired by a variety of classic rhythm games and plans to expand on them by being focused on real-time competitive play.

Author: applied

### How is Quaver different from other games?

> The game is fully open-source, allowing anyone to help and improve the game. Furthermore, we are aiming to provide a new VSGR experience by bringing competitive concepts, such as competitive matchmaking.

Reviewer: The game is fully open-source, which allows*

VSRG*

Author: applied and oops

### How far in development is Quaver?

Quaver is still in early development and is currently in closed alpha. In order to use online features of Quaver, you need to sign up for the alpha.

### What platforms will Quaver be released on?

Quaver will be released on Windows, Mac and Linux.

### Where can I sign up for the Quaver Alpha?

Registrations for the closed alpha occur in rounds. Each round of sign-ups are announced before they open.

### Can I play without being part of the Quaver Alpha?

Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features including online rankings, map uploading, and other community-based features such as in-game chat.

### Does this mean I can’t download maps from the Quaver website?

You cannot download maps from the Quaver website, however you can still download packs via a [mirror](https://rhythmgamers.net/pack/).

### Is there an estimate for the public release date?

We do not have an estimate for the public release date yet.

> 

Reviewer: Remove the extra line break here

Author: Also removed the line break above Troubleshooting.

## Gameplay

### What game modes does Quaver support?

Quaver currently only supports 4 keys (4k) and 7 keys (7k).

### Can I import maps from other games?

Yes, Quaver currently supports .osz and .sm files. To import them, drag the files into the game window.

### I have a lot of maps in an external game, can I import them all at once?

It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.

### Can I import my skin from another game?

The game doesn't support skin conversions out of the box, however you can download a .osk skin converter [here](https://rhythmgamers.net/QBC/) along with a [tutorial](https://www.youtube.com/watch?v=pWeLbx48NVI).

### I found a bug, where can I report it?

Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.

### I have a feature request, where can I post it?

> Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!

Reviewer: Similar to bug reports, feature requests belong on our....

Author: done
